src/components/navigation-header/index.tsx

```diff
@@ -1,76 +1,158 @@
+import { ReactNode } from 'react'
```

Comment:
This component is very skateboard, and I plan to address most items in a separate PR:

- inline `style`s (I used them because I haven't decided how/if I want to break up the 2 headers into separate components & style files)
- repetitive markup
- links don't have hover styles yet
- logos are placeholder images
- product logos aren't links to `/{productSlug}` yet